docusaurus/docs/Android/02-client/06-guides/10-video-integrations/01-video-100ms.mdx

```diff
@@ -307,7 +307,7 @@ Let’s break down each method one by one:
 - **createVideoCallOrJoin**: This method requests an authentication token to the Stream server and joins a call with a specific `callId` parameter. If the `callId` is null, it will request creating a new call dynamically.
 - **createVideoCall**: This method creates a new call dynamically with a unique UUID for the call id.
 - **getVideoCallToken**: This method requests an authentication token to the Stream server.
-- **joinRoom**: Join a room and connect to a video call with the specific name of the room and an authentication token. You can join the room by invoking the `join` method of the `HMSSDK` and `HMSConfig`. For more details, check out the [100ms Join Room](​​https://www.100ms.live/docs/android/v2/features/Join) Android guide.
+- **joinRoom**: Join a room and connect to a video call with the specific name of the room and an authentication token. You can join the room by invoking the `join` method of the `HMSSDK` and `HMSConfig`. For more details, check out the [100ms Join Room](https://www.100ms.live/docs/android/v2/features/Join) Android guide.
```

Comment:
Also, it seems the link address has been changed as below:

https://www.100ms.live/docs/android/v2/features/join